TOPdeskPS/functions/Supporting Files/Get-TdOperator.ps1

```diff
@@ -0,0 +1,41 @@
+﻿function Get-TdOperator {
+    <#
+.SYNOPSIS
+    returns list of operators
+.DESCRIPTION
+    returns list of operators
+.PARAMETER Name
+    human readable name to filter for operator by. Uses the dynamcName field
+.EXAMPLE
+    PS C:\> Get-TdOperator
+    returns list of operators
+#>
+    [CmdletBinding(HelpUri = 'https://andrewpla.github.io/TOPdeskPS/commands/Get-TdOperator')]
+
+    param
+    (
+        [string]
+        $Name
```

Comment:
add an example that shows off using Name parameter.

Reply:
(y)